src/snek5000_cbox/cbox.usr.f

```diff
@@ -103,24 +103,33 @@ subroutine userchk
       endif
 
       ! history points
-      if (mod(ISTEP,nit_hist).eq.0) then
-         if (.not. ifpert) then
-             call hpts()
-         else
 
-             call opcopy(vtmp(1,1),vtmp(1,2),vtmp(1,ndim),vx,vy,vz)
-             n = NX1*NY1*NZ1*NELV
-             call copy(ttmp,T,n)
-             call opcopy(vx, vy, vz, vxp, vyp, vzp)
-             call copy(T,TP,n)
+      nxyz1 = nx1*ny1*nz1
```

Comment:
What are these variables ? Could you please add comments to make this code more understandable?

Reply:
We have `PnPn` and `PnPn-2` formulations. Therefore, the number of grid points is different. I'll add some comments.